common/api/reality-data-client.api.md

```diff
@@ -139,9 +158,18 @@ export class RealityDataClient extends WsgClient {
     getRealityDataRelationships(requestContext: AuthorizedClientRequestContext, projectId: string, realityDataId: string): Promise<RealityDataRelationship[]>;
     getRealityDatas(requestContext: AuthorizedClientRequestContext, projectId: string | undefined, queryOptions: RealityDataRequestQueryOptions): Promise<RealityData[]>;
     getRealityDataUrl(projectId: string | undefined, tilesId: string): Promise<string>;
+    // @public
+    queryRealityData(accessToken: AccessToken, criteria: RealityDataQueryCriteria): Promise<ContextRealityModelProps[]>;

```

Comment:
Why is queryRealityData not defined as part of the RealityDataAccess interface?

Reply:
It was decided that the interface should only enforce what is required for code within the repo to operate. I believe the intent is that the queryRealityData functionality may be available from the provided RealityDataAccess implementation, but we should not be the ones enforcing it.
@calebmshafer is that accurate?

Reply:
Yes, that's correct. We did not want to add anything to the interface that is not explicitly needed by the core-frontend code. That doesn't mean that the client shouldn't have this method and then can be directly used in display-test-app or any other app just like it is today.

The requirements for the client itself will be different, and larger in scope, than what we expect from the RealityDataAccess which is the minimum set we're requiring to create an iTwin.js app (aka IModelApp).